app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -28,7 +28,17 @@ def find_featured_story(stories)
       end
 
       def published_articles_by_tag
-        articles = @tag.present? ? Tag.find_by(name: @tag).articles : Article
+        articles =
```

Comment:
This is a total nitpick, so feel free to ignore but should we maybe add a small comment about the nature of this experiment, how long we expect it to last, etc.?

Reply:
@citizen428 For long-running feature flags, I don't mind doing this, but I don't expect it to be here through the end of the week. 🙂 As soon as we can confirm whether it's a performance improvement, the flag will be removed and the test code will be reverted to its original state.

Reply:
Makes perfect sense, if it's this short-lived I wouldn't add a comment either. Thanks for clarifying!